symbols/arm9.yml

```diff
@@ -3,14 +3,22 @@ arm9:
     - EU
     - NA
     - JP
+    - EU-ITCM
+    - NA-ITCM
   address:
     EU: 0x2000000
     NA: 0x2000000
     JP: 0x2000000
+    EU-ITCM: 0x1FF8000
+    NA-ITCM: 0x1FF8000
   length:
     EU: 0xB7D38
     NA: 0xB73F8
     JP: 0xB8CB8
+    EU-ITCM: 0x4084
```

Comment:
Yeah, the ITCM is 0x8000 bytes long, but the arm9 section of it is only 0x4000 bytes long. Since this is what is being represented in this file, I think that's the size we should use. In the case of ram.yml, the file represents the contents of all the documented symbols in the RAM, so it makes sense to specify the RAM's size.

If we specify 0x8000, declaring a symbol at, say, 0x1FF8000+0x5000 would be considered valid since it's within the specified bounds. But I think a declaration like this should throw an error since that address is outside of the special area.

Reply:
You make a good point. I'm convinced. Let's keep it at 0x4000.